src/backend/history.js

```diff
@@ -1,12 +1,11 @@
 export function trackHistory(hook, bridge) {
   const repo = hook.repo
-  const getSnapshot = () => JSON.stringify(repo.history)
 
-  // Deal with multiple backend injections
-  hook.off('history:reconcile')
+  repo.history.on('reconcile', function() {
+    bridge.send('history:reconcile', JSON.stringify(repo.history))
+  })
 
-  // application -> devtool
-  hook.on('history:reconcile', mutation => {
-    bridge.send('history:reconcile', getSnapshot())
+  repo.history.on('release', function() {
+    bridge.send('history:release', JSON.stringify(repo.state))
   })
 }
```

Comment:
ah yeah, i wonder. my vote is to leave it out for now until we see a need for it. which begs the question if we need the hook at all since it just serves as a way to access `hook.repo`